pcdsdevices/pseudopos.py

```diff
@@ -54,14 +56,15 @@ def calc_combined(self, real_position):
 
         Parameters
         ----------
-        real_position: `namedtuple`
-            The positions of each of the real motors, accessible by name
+        real_position : `~typing.NamedTuple`
```

Comment:
Does this end up linking to real python docs?

Reply:
Yep :)

Reply:
Except it shouldn't have backticks :/
Fixing...